ngi_pipeline/conductor/launchers.py

```diff
@@ -66,7 +76,31 @@ def launch_analysis(projects_to_analyze, restart_failed_jobs=False,
         except (RuntimeError, KeyError, CharonError) as e: # BPA missing from Charon?
             LOG.error('Skipping project "{}" because of error: {}'.format(project, e))
             continue
+        try:
```

Comment:
Yeah the try is there just in case the module throws an exception, because I want to report that there was an error but I don't want the script to die.